### PR TITLE
Adds PIC flag for building shared libs

### PR DESCRIPTION
**Pull request by jkhoogland**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -170,6 +170,10 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU" OR
 	set(CMAKE_CXX_FLAGS_MINSIZEREL "-Os")
 	#
 	set(GCC_EXTRA_OPTIONS "")
+        #
+        if( ${BUILD_SHARED_LIBS} )
```
Comment (jbeder) [inline CMakeLists.txt]: This can just be

if(BUILD_SHARED_LIBS)

Also, please make sure the indent is correct.

**Comment by jbeder** [comment]
Please rebase this against HEAD so that there is only one commit in this pull request.

Also, can you update the commit description:

1) Write in imperative tense ("Add PIC flag...").
2) Put short description of the reason for this change.

**Comment by jkhoogland** [comment]
Modified as requested.

**Comment by jbeder** [comment]
Merged 4376ebacaad3d740ad95545b01084ddf545bb5f4.

**Pull request closed without merging**